exports.js

```diff
@@ -1038,7 +1038,10 @@ module.exports = {
         'amsDiagnosticLogsEnabled'      : require(__dirname + '/plugins/azure/mediaServices/amsDiagnosticLogsEnabled.js'),
         'amsPublicAccessDisabled'       : require(__dirname + '/plugins/azure/mediaServices/amsPublicAccessDisabled.js'),
         'amsManagedIdentityEnabled'     : require(__dirname + '/plugins/azure/mediaServices/amsManagedIdentityEnabled.js'),
-        'amsClassicApiDisabled'         : require(__dirname + '/plugins/azure/mediaServices/amsClassicApiDisabled.js')
+        'amsClassicApiDisabled'         : require(__dirname + '/plugins/azure/mediaServices/amsClassicApiDisabled.js'),
+
+        'automationAccountDiagnosticLogs': require(__dirname + '/plugins/azure/automationAccounts/automationAccountDiagnosticLogs.js'),
+        'automationAccountManagedIdentity': require(__dirname + '/plugins/azure/automationAccounts/automationAccountManagedIdentity.js'),
```

Comment:
```suggestion
        'automationAccountDiagnosticLogs': require(__dirname + '/plugins/azure/automationAccounts/automationAccountDiagnosticLogs.js'),
        'automationAccountManagedIdentity': require(__dirname + '/plugins/azure/automationAccounts/automationAccountManagedIdentity.js'),
```
can we make the names shorter?